master/lustregittagpoller.py

```diff
@@ -0,0 +1,224 @@
+# -*- python -*-
+# ex: set syntax=python:
+
+import itertools
+import os
+import re
+import urllib
+
+from twisted.internet import defer
+from twisted.internet import utils
+from twisted.python import log
+
+from buildbot import config
+from buildbot.changes import base 
+from buildbot.util import epoch2datetime
+from buildbot.util.state import StateMixin
+
+class LustreTagPoller(base.PollingChangeSource, StateMixin):
+
+    """ This source will poll a remote git repo for
```

Comment:
The first line of the comment should not span two lines:

"Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line."